java/sage/Wizard.java

```diff
@@ -15,66 +15,34 @@
  */
 package sage;
 
-import static sage.SageConstants.LITE;
-
 import org.apache.lucene.analysis.Analyzer;
 import org.apache.lucene.document.Document;
 import org.apache.lucene.document.Field;
-import org.apache.lucene.index.CorruptIndexException;
-import org.apache.lucene.index.IndexCommit;
-import org.apache.lucene.index.IndexReader;
-import org.apache.lucene.index.IndexWriter;
-import org.apache.lucene.index.IndexWriterConfig;
+import org.apache.lucene.index.*;
```

Comment:
OK, it changed my comment when I put a "dot star" and apparently inferred that to mean italics...odd. I'm totally fine with imports, its just the wildcard ones I'm not a fan of.